egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -66,6 +66,7 @@ clipping_threshold=30     # if norm_based_clipping is true this would be the max
                           # else this is the max-absolute value of each element in Jacobian.
 chunk_width=20  # number of output labels in the sequence used to train an LSTM
 chunk_left_context=40  # number of steps used in the estimation of LSTM state before prediction of the first label
+chunk_right_context=40  # number of steps used in the estimation of LSTM state before prediction of the first label
```

Comment:
default chunk_right_context has to be 0, as the default recipe is LSTM. If you want you can can run the make_configs.py script to figure out if the arch. has a BLSTM and use a new set of defaults in this case,.